docs/docker-hub/userguide.md

```diff
@@ -33,7 +33,7 @@ you can look at your profile page on [Docker Hub](https://hub.docker.com).
 
 ## Pulling, running and building images
 
-You can find more information on [working with Docker images](../userguide/dockerimages.md).
+You can find more information on [working with Docker images](/userguide/dockerimages/).
```

Comment:
This is weird; I placed this comment (about the tabs) on the `docs/touch-up.sh` script! Probably a GitHub glitch.

Anyway, this is what I was talking about;

![screen shot 2015-07-25 at 00 20 58](https://cloud.githubusercontent.com/assets/1804568/8885814/5f21f18e-3263-11e5-8f48-a696ad0cd2a9.png)